vscode/src/non-stop/FixupController.ts

```diff
@@ -877,7 +877,7 @@ export class FixupController
             return
         }
         const previousRange = task.selectionRange
-        const previousInstruction = task.instruction
+        const previousInstruction = task.instruction.replace(/^\/edit/, '')
```

Comment:
```suggestion
        const previousInstruction = task.instruction.replace(/^\/edit /, '')
```

Good catch! I'm thinking maybe instead of the `retry` step, we should do this during the FixupTask creation step to make sure /edit is removed for all tasks, what do you think? @deepak2431 @toolmantim

Reply:
I'm not quite sure what the trade off is here, but trust your call @abeatrix!

Reply:
@abeatrix Agree with you, that would be good to do I think so.

Reply:
Oh woops I meant to tag Tom @umpox  here, sorry @toolmantim lol